src/main/java/ui/command/TaskListCommand.java

```diff
@@ -0,0 +1,21 @@
+package ui.command;
+
+import data.TaskList;
+
+/**
+ * Command which operates on the TaskList
+ * maintained by ChatBot
+ */
+abstract class TaskListCommand extends Command {
```

Comment:
In your view, do you think this class can be combined with the Command class?

Reply:
It does seem that this class seems to be redundant now. It was initially used to separate commands that required access to the task list and those that did not, but that separation has not much purpose.